NEWS.md

```diff
@@ -1,6 +1,7 @@
 ## 18.2.0 IN-PROGRESS
 
 * Upgrade to RMB 33.1.3 and Folio Service Tools 1.7.2. (CVE-2021-44228) (MODUSERS-295)
+* Upgrade to RMB 33.2.4 (MODUSERS-301)
```

Comment:
I would combine these two lines. I don't think it is necessary to describe RMB upgrades twice in the release news

```suggestion
* Upgrade to RMB 33.2.4 and Folio Service Tools 1.7.2. (CVE-2021-44228) (MODUSERS-295, MODUSERS-301)
```